docs/_interface/consensus-clients.md

```diff
@@ -0,0 +1,149 @@
+
+A complete command to start Geth so that it can connect to a consensus client looks as follows:
+
+```shell
+geth --authrpc.addr localhost --authrpc.port 8551 --authrpc.vhosts localhost --authrpc.jwtsecret /tmp/jwtsecret
+```
+
+
+## Consensus clients
+
+There are currently five consensus clients that can be run alongside Geth. These are:
+
+[Teku](https://pegasys.tech/teku): written in Java
+ 
+[Nimbus](https://nimbus.team/): written in Nim
+ 
+[Lighthouse](https://lighthouse-book.sigmaprime.io/): written in Rust
+ 
+[Lodestar](https://lodestar.chainsafe.io/): written in Typescript
+ 
+[Prysm](https://docs.prylabs.network/docs/getting-started/): written in Go
+
+There is currently a [client diversity][client-div-link] issue where a large dominance of Prysm clients poses a risk 
+to the health of the network. In response to the initial drive to even out the client diversity many Prysm nodes switched 
+to Lighthouse to the extent that it now also has a problematic market share. It is therefore recommended to use a minority 
+client such as Teku, Nimbus or Lodestar (although users should be aware that Lodestar is currently awaiting a third-party audit). 
+
+It is also worth noting that client diversity is also an [issue on the execution layer](https://clientdiversity.org/), 
+with Geth being run on about 85% of all Ethereum nodes. It is assumed that readers on this page have at 
+least considered running a [minority execution client][execution-clients-link] and concluded that Geth is the right 
+client for them.
+
+There are several ways to download and install the consensus clients including prebuilt binaries, docker containers
+or building from source. Instructions for each client are provided in the documentation linked in the client list above.
+Users can choose the method that is right for them.
+
+Regardles of the installation method, the consensus client must be started with the right port configuration to 
+establish an RPC connection to the local Geth instance. In the example above, `localhost:8551` was authorized 
+for this purpose. The consensus clients all have a command similar to `--http-webprovider` that 
+takes the exposed Geth port as an argument.
+
+The consensus client also needs the path to Geth's `jwt-secret` in order to authenticate the RPC connection between them.
+Each consensus client has a command similar to `--jwt-secret` that takes the file path as an argument. This must
+be consistent with the `--authrpc.jwtsecret` path provided to Geth.
+
+The consensus clients all expose a [Beacon API][beacon-api-link] that can be used to check the status
+of the Beacon client or download blocks and consensus data by sending requests using tools such as [Curl](https://curl.se).
+More information on this can be found in the documentation for each consensus client.
+
+## Validators
+
+After The Merge, miners are n longer responsible for securing the Ethereum blockchain. Instead, this becomes the responsibility
```

Comment:
*After The Merge, miners are _no_ longer responsible for securing the Ethereum blockchain. Instead, this becomes the responsibility

Reply:
thanks for the catch